katdal/vis_flags_weights.py

```diff
@@ -146,7 +153,7 @@ def corrprod_to_autocorr(corrprods):
 
 
 @numba.jit(nopython=True, nogil=True)
-def weight_power_scale(vis, weights, auto_indices, index1, index2, out=None):
+def weight_power_scale(vis, weights, auto_indices, index1, index2, divide=True, out=None):
```

Comment:
Maybe put `divide` at the end, so that it won't break code that passes `out` positionally (it looks like katsdpcal passes it by keyword so it is probably fine).

Reply:
I was going to leave it as is, using the lack of users and NumPy's API as excuse, but I now see that e.g. `np.multiply` does not have it as the final argument either... So I've moved it to the end.